app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -246,7 +247,7 @@ fun questTypeRegistry(
     9 to AddCarWashType(),
 
     10 to AddBenchBackrest(),
-    11 to AddAmenityCover(featureDictionaryFuture),
+    11 to AddAmenityCover { featureDictionaryFuture.get().getFeature(it) },
```

Comment:
Looks like the function should be passed as a parameter of `questTypeRegistry`, too.

Reply:
Done in [`af75a4c`](https://github.com/streetcomplete/StreetComplete/pull/5273/commits/af75a4ca3e1dc7f58ab61a2a9c64c6867c039dc8).